src/cmd.ts

```diff
@@ -0,0 +1,26 @@
+import { ChildProcessByStdio, spawn } from "child_process"
+
+export async function run_yarn(targetDir: string): Promise<void> {
+  const childProcess = spawn("yarn", ["--cwd", targetDir], {
+    stdio: [process.stdin, process.stdout, process.stderr],
+  })
```

Comment:
Oh neat, I didn't know about yarn's `--cwd` option! Another more general way to do this is to use spawn's `cwd` option:
```ts
  const childProcess = spawn("yarn", {
    cwd: targetDir,
    stdio: [process.stdin, process.stdout, process.stderr],
  })
```